packages/salesforcedx-vscode-apex-debugger/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { DebugClient } from 'vscode-debugadapter-testsupport';
```

Comment:
OK. @guw should take a look at how to get the encrypted jwt keys onto Travis and Appveyor following https://github.com/wadewegner/sfdx-travisci (not sure how to do it for AppVeyor) so that we can get some integration tests up (not just for debugger but for other things)